CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Add an achievement badger to the PR ([#3721](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3721))
 - Upgrade the backport workflow ([#4343](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4343))
 - [Lint] Add custom stylelint rules and config to prevent unintended style overrides ([#4290](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4290))
+- [Lint] Add stylelint rule to decline configured properties ([#4374](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4374))
```

Comment:
Can we rephrase this to be clearer? It seems like the change is that we can now declare CSS properties that can only be specified by allowlisted files. Something along those lines would be better for someone scanning the release notes without context.